lms/djangoapps/courseware/tabs.py

```diff
@@ -42,12 +42,12 @@ class CoursewareTab(EnrolledTab):
 
     def __init__(self, tab_dict):
         def link_func(course, reverse_func):
-            if course_home_mfe_is_active(course.id):
-                return get_learning_mfe_home_url(course_key=course.id, view_name='home')
-            else:
+            if course_home_legacy_is_active(course.id):
```

Comment:
Oooh, I appreciate you keeping the truthy if statement rather than falsy (i.e. `if not course_home_legacy_is_active`) 👏